src/hooks/usePanel.tsx

```diff
@@ -6,10 +6,11 @@ import { usePanelPortal } from "hooks/usePanelPortal";
 
 export const usePanel = <D extends object>(
   getPanel: (closePanel: () => void, data: D | null) => ReactNode,
+  isMedium: (data: D | null) => boolean = () => true,
```

Comment:
I've added the ability to change the width from within panels, so this function won't be necessary once this PR lands: https://github.com/canonical/rebac-admin/pull/97.